robot_skills/src/robot_skills/robot.py

```diff
@@ -290,10 +288,11 @@ def __exit__(self, exception_type, exception_val, trace):
             rospy.logerr("Robot exited with {0},{1},{2}".format(exception_type, exception_val, trace))
         self.close()
 
+
 def _collect_needs_desires(needs, desires, test_func):
     """
     :param needs: Collection of needed values. None means nothing is needed.
-    :param desireds: Collection of desired values, None means nothing is desired.
+    :param desires: Collection of desired values, None means nothing is desired.
```

Comment:
Actually, `desireds` was intended, as in the collection of desired things (just like `needs` is the collection of needed things).